convert2rhel/actions/pre_ponr_changes/backup_system.py

```diff
@@ -77,8 +81,51 @@ def run(self):
 
         super(BackupRepository, self).run()
 
-        repo.backup_yum_repos()
-        repo.backup_varsdir()
+        self.backup_yum_repos()
+        self.backup_varsdir()
```

Comment:
Maybe we could create a new action class just for this guy here... It is not really a repository, but a variable that is used by the repositories.

Reply:
Okay, makes sense! I'll change that